neurom/check/neuron_checks.py

```diff
@@ -278,14 +283,15 @@ def is_dangling(neurite):
         if neurite.type != NeuriteType.axon:
             return True
 
-        all_points = list(chain.from_iterable(_neurite.points[1:] for _neurite in iter_neurites(n)
-                                              if _neurite.type != NeuriteType.axon))
+        all_points = list(chain.from_iterable(n.points[1:]
+                                              for n in iter_neurites(neurite)
+                                              if n.type != NeuriteType.axon))
         res = [np.linalg.norm(starting_point - p[COLS.XYZ]) >= 2 * p[COLS.R] + 2
                for p in all_points]
         return all(res)
 
-    bad_ids = [(neurite.root_node.id, [neurite.root_node.points[1]])
-               for neurite in iter_neurites(n) if is_dangling(neurite)]
+    bad_ids = [(n.root_node.id, [n.root_node.points[1]])
```

Comment:
It should be 'n' (the new one) -> neurite
neurite (the new one) -> neuron

Reply:
yup, thanks